diff --git a/src/ds5/ds5-factory.cpp b/src/ds5/ds5-factory.cpp
index 7834df0..f735fa2 100644
--- a/src/ds5/ds5-factory.cpp
+++ b/src/ds5/ds5-factory.cpp
@@ -578,6 +578,7 @@ namespace librealsense
             }
 
 
+#if !defined(ANDROID) && !defined(__APPLE__) // Not supported by android & macos
             auto is_pid_of_hid_sensor_device = [](int pid) { return std::find(std::begin(ds::hid_sensors_pid), std::end(ds::hid_sensors_pid), pid) != std::end(ds::hid_sensors_pid); };
             bool is_device_hid_sensor = false;
             for (auto&& uvc : devices)
@@ -592,6 +593,7 @@ namespace librealsense
             {
                 all_sensors_present &= (hids.capacity() >= 2);
             }
+#endif
 
             if (!devices.empty() && all_sensors_present)
             {
